components/library/tntnet/tntnet-demos.p5m

```diff
@@ -0,0 +1,153 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Jim Klimov
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)/libraries@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
```

Comment:
hmmm... a mis-merge between workspaces again? :(
I remember fixing this somewhere...